connector/src/main/resources/config.yml

```diff
@@ -13,6 +13,9 @@ bedrock:
   address: 0.0.0.0
   # The port that will listen for connections
   port: 19132
+  # Clone remote server port number
+  # Geyser will try to set bedrock port to the same as java's server port if it is run as a plugin
```

Comment:
I would change this language to something like:
```
# Use Java server port
# This option ignores the above port and uses the Java server's port as Bedrock's port. Only available for plugin versions.

Reply:
I've made the edit, copied uwu's desc.

Reply:
OwO?! Someone said my name